style.css

```diff
@@ -37,17 +38,19 @@ label {
   box-sizing: border-box;
 }
 
-html,
 body {
   margin: 0;
   font-family: "Poppins", sans-serif;
   line-height: 1.5em;
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  grid-template-columns: auto;
   color: #172b4d;
-  background-image: url("./header-shapes mobile.png");
+  background-image: url("./images/header-shapes mobile.svg");
```

Comment:
- [ ] Please note that this background image should only be for the headline section (that is the first section after the header), not on your HTML body. Kindly remove it from the HTML body and add it to the headline section only so that it will not display over all other elements.
### Figma headline section.
![image](https://user-images.githubusercontent.com/94551484/227714228-af572e2d-1559-456e-9c7d-b6c9f538c2ad.png)
### Your design displaying over all sections
[Screencast from 25-03-2023 12:16:20.webm](https://user-images.githubusercontent.com/94551484/227714283-c9b351ab-1568-46f9-b8c4-f19c764ae70a.webm)